package.json

```diff
@@ -349,7 +349,7 @@
     "dedent": "^0.7.0",
     "delete-empty": "^2.0.0",
     "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.9.0",
+    "enzyme-adapter-react-16": "^1.10.0",
```

Comment:
Upgrade enzyme-adapter-react-16 to support React.memo
https://github.com/airbnb/enzyme/pull/1914